packages/frontend/src/pages/Send/useSendData.tsx

```diff
@@ -31,7 +31,7 @@ const useSendData = (
     {
       enabled:
         !!token?.address && !!fromNetwork?.slug && !!toNetwork?.slug && !!fromAmount?.toString(),
-      refetchInterval: 5e3,
+      refetchInterval: 10e3,
```

Comment:
it looks like the intervals vary in components; ie `5`, `7`, or `10` seconds. maybe this can be more consistent throughout